dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExport.java

```diff
@@ -104,6 +104,10 @@ public class KubernetesExport extends Export {
                         description = "The image builder used to build the container image (e.g. docker, jib, podman).")
     protected String imageBuilder;
 
+    @CommandLine.Option(names = { "--image-push" }, defaultValue = "true",
```

Comment:
Should it be set to true in the export command ?

Reply:
In the export it sets the property in the Maven pom.xml and prepares the Maven project with image push enabled. I would say it makes sense to have it set to true

Reply:
I am mixing with run command, my bad :)